_vendors.yml

```diff
@@ -172,3 +172,13 @@ creative_commons:
   version: 2020.11.3
   dir: assets/license_badges
   alias: creativecommons-vocabulary
+wavedrom:
+  name: 'wavedrom'
+  version: 3.2.0
+  file: wavedrom.min.js
+  integrity: sha512-/ZL0uQxVV1wYyWlpO4klZ1a39eaBz4zESSamuBMaMsZ6le3YejJ07hmLlHoCTXrKz5eYtEuO5K1BcTo+lQpQJA==
+wavedrom_skin:
+  name: 'wavedrom'
+  version: 3.2.0
+  file: skins/default.min.js
```

Comment:
`skins/default.min.js` does not exist in its npm version. You need to change it to `skins/default.js` and re-calculate its integrity value

Reply:
got it. moved to default.js now.